Documentation/ceph-common-issues.md

```diff
@@ -34,7 +34,7 @@ After you verify the basic health of the running pods, next you will want to run
 
 - Logs on a specific node to find why a PVC is failing to mount:
   - Rook agent errors around the attach/detach: `kubectl logs -n rook-ceph <rook-ceph-agent-pod>`
-- See the [log collection topic](ceph-advanced-configuration.md#log-collection) for a script that will help you gather the logs
+- See the [log collection topic](ceph-advanced-configuration.md#log-collection) for a script that will help you gather the logs.
```

Comment:
The trend I see in most of the rest of the docs and the guideline I see most often is to make sure bulleted list items do *not* end in a period.